addons/binding/org.openhab.binding.squeezebox/README.md

```diff
@@ -61,3 +61,59 @@ All devices support some of the following channels:
 | coverartdata            | Image        | Image data of cover art of the current song|
 | ircode                  | String       | Received IR code|
 | numberPlaylistTracks    | Number       | Number of playlist tracks|
+| notificationSoundVolume | Dimmer       | Volume for playing notifications|
+
+## Notifications
+
+Squeeze Players can be set up as audio sinks in openHAB
+
+### How To Set Up
```

Comment:
I would remove both sections "How To Set Up" and "How To Use" - none of this is specific to Squeezebox, but valid for all available audio sinks. So instead of adding such info here, the general documentation should be enhanced (at http://docs.openhab.org/configuration/multimedia.html).